gtest/src/runner.rs

```diff
@@ -126,14 +126,16 @@ pub fn init_fixture<SC: StorageCarrier>(
     storage: Storage<SC::PS>,
     test: &Test,
     fixture_no: usize,
-) -> anyhow::Result<(WasmRunner<SC>, Vec<Message>)> {
+) -> anyhow::Result<(WasmRunner<SC>, Vec<Message>, Vec<Message>)> {
+    let storage2 = storage.clone();
```

Comment:
this is a subject to change so simple naming

Reply:
Why to clone?

Reply:
Ah, I see, `storage` has been consumed by `run`. Need to be refactored but within other PRs.